### ISSUE TITLE
Multiple (redundant?) return statements in test_jit.py

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#22109]
In https://github.com/pytorch/pytorch/blob/master/test/test_jit.py#L5900-L5945 there are several methods with multiple return statements like below:

```python
@torch.jit.script_method
def forward(self, x):
    x = self.conv(x)
    return x
    x = self.bn(x)
    return F.relu(x, inplace=True)
```

It seems they are redundant, as subsequent return statements are unreachable. If they are there for a debugging purpose, a comment would be nice.

**Comment by apaszke** [comment]
Looks like a bug to me 😕

**Comment by vadimkantorov** [comment]
@apaszke on a side note, if I want to do a fused inplace Dropout+Relu, could jit be of help in its current state? (I understand I could do a custom autograd Function to sidetrack tensor versioning issues)

**Comment by apaszke** [comment]
On a side note I think we're not generating in-place kernels at the moment, but it wouldn't be too hard to adjust our codegen to take care of that. We might be able to automatically change them to out-of-place ops and fuse them at that point even now, but idk if we have merged this feature in the end (I worked on it some time ago).
